src/client/packaging/pypi/CHANGELOG.md

```diff
@@ -3,6 +3,31 @@
 All notable future changes to the `delphi_epidata` python client will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/).
 
+## [4.1.17] - 2024-01-30
```

Comment:
you forgot to re-add the link to #1418 and that functionality should be under the pending 4.1.21 heading